### ISSUE TITLE
diann_maxlfq {diann} for multiplex-DIA

### ISSUE THREAD

**Issue by Li5459000** [vdemichev/DiaNN#738]
Hi Vadim,

I'm attempting to use R to process the main report generated by DIANN in multiplex mode, but I'm a beginner in R. 

The code in the diann-rpackage seems to be designed for LF-DIA only?
What modifications do I need to make to the diann-rpackage code to make it compatible with multiplex-DIA's main report?

**Comment by vdemichev** [comment]
Yes, the package is for label-free. For multiplexing, need to write custom scripts to quantify proteins. Unfortunately, no way around that at the moment.
